### ISSUE TITLE
Play Store Age restriction

### ISSUE THREAD

**Issue by bytecorner-jan** [vector-im/element-android#4821]
#### What would you like to do?

I'm trying to set up a Tablet for my child and want to install Element app for him. I "secure" this with Family link for Child protection. Therefore, the age restriction of 18 years in the play store is causing issues. I'm unable to install element in the child's account

#### Why would you like to do it?

I would like to take advantage of all the age safety measures Family link provides, but also use Element with my child. Hence I have to use the correct age. 

#### How would you like to achieve it?
 I dont't see why element is restricted above 18 year.  No other messenger Whatsapp Threema ... seems to have this restriction.

**Comment by bmarty** [comment]
Element can give access to content for adults only, that's why on some countries the Google PlayStore requires that the user's age is above 18.

2 possible alternatives:
- install Element by downloading the APK from the GitHub release: https://github.com/vector-im/element-android/releases, but with the limitation that you will not get the updates automatically.
- install the F-Droid version from https://f-droid.org/packages/im.vector.app/, I do not think there is age limitation there (not 100% sure though). This version does not use Push so can consume more battery, but for the rest this is the same app.

**Comment by bytecorner-jan** [comment]
I also tried F-Droid, but this seems not to work. 

The GitHub apk did the trick. Thanks

But still I think it's the wrong sign to send for a messenger app. This also means that element/matrix will never be a solution for e.g. a school. Which, I think, should be possible.

**Comment by kojid0** [comment]
I see your point but Element can't do anything about this, other than releasing an app only for children with content scanning. I read somewhere that this is on their list. So I suggest to close this issue as it has nothing to do with the Android Team themselves.

**Comment by bytecorner-jan** [comment]
@kojid0  thanks for the explanation. 
Could you point me to where you think you read, or where I can start searching? 

Could anyone point me to where this age restriction is defined or where it is applied (app store release process)? 

To understand, this "content scanning" would have to be performed by the app itself? I'm not aware of any other messenger doing such a thing. 

@bmarty: 

Is this a decision from element with the access to adult content? I was just looking at other apps in the playstore: 
Messenger: Whatsapp, Threem, Signal - every age, Telegram: 17
Browser: firefox, edge, chrome (definitively access to adult content possible) - every age
other: tiktok 13, snapchat 13, instagram 13
Facebook (meta): Messenger - every age, instagramm 13, facebook 13

As in Germany there seems to be pushback for Microsoft solutions in the school usage coming, at least the state I live in Baden-Württemberg, I would see matrix/element as an alternative

I would like to help on this issue, so any hint to where read, or whom to talk to is very much welcome.

**Comment by kojid0** [comment]
> Could you point me to where you think you read, or where I can start searching?

Content Scanning (no definition what this is for) is on their public roadmap: https://github.com/vector-im/roadmap/projects/1#card-49300883

> Could anyone point me to where this age restriction is defined or where it is applied (app store release process)?

Maybe this helps: https://support.google.com/googleplay/android-developer/answer/9859655?hl=en

> To understand, this "content scanning" would have to be performed by the app itself? I'm not aware of any other messenger doing such a thing.

I don't know. Just speculations.

**Comment by semisenex** [comment]
Forked versions do not have an age restriction in Germany.
https://play.google.com/store/apps/details?id=im.synod
https://play.google.com/store/apps/details?id=de.spiritcroc.riotx

**Comment by toby63** [comment]
@bmarty 

I would like to request a reevaluation of this.

According to [this official info from Google Play Store](https://support.google.com/googleplay/answer/6209544?visit_id=1659295314722-853850986592113990&p=appgame_ratings&rd=1) you rate the App yourself. Is that correct?

So I don't see why you do this.
Age restrictions are not for what could possibly be included in an app, age restrictions are for apps which really actually include specific content like violence etc.

As it was already pointed out even Firefox etc. are rated for all ages, even though you can clearly use it to browse the whole internet.
And I don't see a natively implemented content restriction option (e.g. for kids) in Firefox, so the lack if this feature in Element can't be the reason.

I would like to suggest giving this app at least a much lower rating, so an installation without needing a specific adult verification process (for 18+ apps) is made possible.

**Comment by pwFoo** [comment]
Following here...
Tried to install my child Element to replace WhatsApp... WhatsApp work, Element not because of age restriction...

**Comment by Pavouk106** [comment]
I have installed Element on my daughter's phone, she has PEGI12 restriction on her account (set through Family Link) and I had to approve the installation as a parent. This was around start of August 2023, I had no issues.

Now her classmates wanted to get in contact with her and they can't install, actually even find (I can if I use the exact full name), Element in Google Play because it got rated PEGI18 as it contains (and this is directly from Google Play page) "extreme violence". It can't be installed on the phone even if I remove restrictions set by Family Link and I approve the install as a parent - it says this app can't be installed on the account of someone under 18 years. Yet my daughter has this very app installed. So it seems it was rated below PEGI18 (haven't checked back then when I installed it) and got re-rated in the august/september.

**If somebody who can do it reads this - please go through the evaluation once again to de-restrict it.**

As others pointed out - web browsers that allows you (and kids) to browse the whole internet are PEGI3, there is absolutely no reason for Element to be PEGI18.

If I can help by any means, let me know. Even if it is as small as writing my own perspective of the app to Google or if there is some way to appeal the rating as ordinary user. Just let me know how and I will happily do that.

For now I can sideload Element either directly from here or via F-Droid. Does F-Droid still lack Push (as mentioned here in this thread)? I installed it from F-Droid on another device to test it and it got picked up by Google Play (uninstall and open buttons are present in Google Play) - will Google Play update it?

**Comment by huguesdk** [comment]
i ended up in this issue trying to report the obviously erroneous age rating that element android currently has. it is set at pegi 18, and the description says:

> Extreme Violence
> Users Interact, Shares Location

i think this is clearly wrong (especially when compared to other messaging apps). i suspect that this may be a result of sabotage to harm element’s reputation.

for comparison, here is the current rating of other popular messaging apps:

*   whatsapp: pegi 3
*   snapchat: parental guidance
*   telegram: parental guidance
*   messenger: parental guidance
*   discord: parental guidance
*   signal: pegi 3
*   google chat: pegi 3
*   slack: pegi 3
*   teams: pegi 3
*   instagram: parental guidance

could someone align the current rating to pegi 3 (like whatsapp and signal), or at least parental guidance?

**Comment by acmartinsalmeida** [comment]
Annoyed too here in France
Curiously [Element X](https://play.google.com/store/search?q=element%20X&c=apps) is Parental Guidance.
Very strange

**Comment by janus007** [comment]
Yes, it seems a bit like someone believe that Element is serious contender against the established apps.

**Comment by MattiJarvinen** [comment]
Oh, ffs. Just bumbed into this. Was going to ditch FluffyCat on kid's phone but no it's not possible.

**Comment by Linuxine** [comment]
Hi ! I am a huge Element fan since many years, and I have achieved to convince many friends and family members to use it instead of Whats'App, because it is simply better at so many levels. Now they are also recommending it to their own friends, and some of them would like to install it on their children's phone, but they cannot due to this mistake in the PEGI rating... 

Obviously this is a mistake from Google, there is no "extreme violence" in Element, and it makes no sense that all the others chat apps like What'App are classified as much less dangerous (PEGI 3 for What'sApp seems a bit low, IMO), whereas the functionalities are the same in Element, minus the personal data sharing with Facebook and co...

Could you try to contact Google about it ? I know the process to achieve to talk to a real human is very painful and time consuming, but it would really help to promote Element to the general public ! Thanks a lot in advance !

**Comment by outofrange** [comment]
> could someone align the current rating to pegi 3 (like whatsapp and signal), or at least parental guidance?

Would this possible?

We run Element / Synapse in our company, but apk sideloading / F-Droid aren't an option for all of our employees, since some have that restricted through device policies from connected Microsoft accounts.
